plugins/services/src/js/reducers/serviceForm/Containers.js

```diff
@@ -456,19 +411,15 @@ module.exports = {
       return container;
     });
 
-    if (field === 'healthChecks') {
-      if (newState[index].healthChecks == null) {
-        newState[index].healthChecks = [];
-      }
-
-      if (this.healthChecks[index] == null) {
-        this.healthChecks[index] = {};
+    if (field === 'healthCheck') {
+      if (this.healthCheckState[index] == null) {
+        this.healthCheckState[index] = {};
```

Comment:
*Note to self: We need to refactor the container reducer as those indexes as well as the crazy number of conditions makes it really hard to follow through.*